src/hooks/apis/users.ts

```diff
@@ -0,0 +1,57 @@
+import { API_PATH } from '@/utils/constants';
+
+export const getUserByEmail = async (email: String) => {
```

Comment:
im noticing that your using String with upper case instead of string lower case which is the primitive type for typescript, is their a reason for that ?

Reply:
https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string

Theyre the same typing